spec/features/draws/draw_start_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw activation' do
+  let(:draw) { FactoryGirl.create(:draw, status: 'draft') }
+
+  context 'as admin' do
+    before { log_in(FactoryGirl.create(:admin)) }
+
+    it 'can be initiated' do
+      create_draw_resources(draw)
+      visit draw_path(draw)
+      click_link('Begin draw process')
+      expect(page).to have_css('.flash-notice',
+                               text: 'Draw successfully initiated.')
+    end
+
+    it 'redirects to show on failure' do
+      visit draw_path(draw)
+      click_link('Begin draw process')
+      # TODO: figure out if there's a better assertion?
+      expect(page).to have_css('.flash-error')
+    end
+
+    def create_draw_resources(draw)
+      FactoryGirl.create_list(:student, 2, draw: draw)
+      draw.suites << FactoryGirl.create(:suite_with_rooms, rooms_count: 2)
+    end
+  end
+
+  context 'as student' do
+    before { log_in(FactoryGirl.create(:student)) }
+
+    it 'cannot see the start button' do
+      visit draw_path(draw)
+      expect(page).not_to have_link('Begin draw process')
+    end
+  end
```

Comment:
I still think we're better off using [view specs](https://robots.thoughtbot.com/how-we-test-rails-applications#view-specs) for these kinds of things